FULLCONNECTION/javascripts/es6/script.js

```diff
@@ -111,7 +111,7 @@ function submitSignup() {
 			document.querySelector('.form_error_text small').textContent = 'Your Passwords do not match';
 		} else {
 			document.querySelector('.form_error_text').style.display = 'none';
-			fetch('http://localhost:3000/api/v1/auth/signup', {
+			fetch('https://kampkelly-mydiary-api.herokuapp.com/api/v1/auth/signup', {
```

Comment:
You can create a baseUrl ('https://kampkelly-mydiary-api.herokuapp.com/api/v1') and then append your route to it.